app/src/main/java/fr/free/nrw/commons/category/CategoryImagesActivity.java

```diff
@@ -112,6 +112,7 @@ public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
         if (mediaDetails == null || !mediaDetails.isVisible()) {
             // set isFeaturedImage true for featured images, to include author field on media detail
             mediaDetails = new MediaDetailPagerFragment(false, true);
+            //FIXME: check if it is even required.
```

Comment:
What is `it`? Have a feeling this is similar to `CategoryDetailsActivity.java`

Reply:
This wasn't supposed to be here. It was just a point for me to check. Will remove.